integrations/server/test_covidcast_endpoints.py

```diff
@@ -185,31 +221,26 @@ def test_correlation(self):
     def test_csv(self):
         """Request a signal from the /csv endpoint."""
 
-        rows = [self._make_placeholder_row(time_value=20200401 + i, value=i)[0] for i in range(10)]
+        rows = [CovidcastRow.make_default_row(time_value=20200401 + i, value=i) for i in range(10)]
         first = rows[0]
         self._insert_rows(rows)
 
         response = requests.get(
             f"{BASE_URL}/csv",
-            params=dict(signal=first.signal_pair(), start_day="2020-04-01", end_day="2020-12-12", geo_type=first.geo_type),
+            params=dict(signal=first.signal_pair, start_day="2020-04-01", end_day="2020-12-12", geo_type=first.geo_type),
         )
-        response.raise_for_status()
-        out = response.text
-        df = pd.read_csv(StringIO(out), index_col=0)
-        self.assertEqual(df.shape, (len(rows), 10))
-        self.assertEqual(list(df.columns), ["geo_value", "signal", "time_value", "issue", "lag", "value", "stderr", "sample_size", "geo_type", "data_source"])
 
     def test_backfill(self):
         """Request a signal from the /backfill endpoint."""
 
         num_rows = 10
-        issue_0 = [self._make_placeholder_row(time_value=20200401 + i, value=i, sample_size=1, lag=0, issue=20200401 + i)[0] for i in range(num_rows)]
-        issue_1 = [self._make_placeholder_row(time_value=20200401 + i, value=i + 1, sample_size=2, lag=1, issue=20200401 + i + 1)[0] for i in range(num_rows)]
-        last_issue = [self._make_placeholder_row(time_value=20200401 + i, value=i + 2, sample_size=3, lag=2, issue=20200401 + i + 2)[0] for i in range(num_rows)] # <-- the latest issues
+        issue_0 = [CovidcastRow.make_default_row(time_value=20200401 + i, value=i, sample_size=1, lag=0, issue=20200401 + i) for i in range(num_rows)]
+        issue_1 = [CovidcastRow.make_default_row(time_value=20200401 + i, value=i + 1, sample_size=2, lag=1, issue=20200401 + i + 1) for i in range(num_rows)]
+        last_issue = [CovidcastRow.make_default_row(time_value=20200401 + i, value=i + 2, sample_size=3, lag=2, issue=20200401 + i + 2) for i in range(num_rows)] # <-- the latest issues
```

Comment:
nits:
```suggestion
        issue_0 = [CovidcastRow.make_default_row(time_value=2020_04_01 + i, value=i, sample_size=1, lag=0, issue=2020_04_01 + i) for i in range(num_rows)]
        issue_1 = [CovidcastRow.make_default_row(time_value=2020_04_01 + i, value=i + 1, sample_size=2, lag=1, issue=2020_04_01 + i + 1) for i in range(num_rows)]
        last_issue = [CovidcastRow.make_default_row(time_value=2020_04_01 + i, value=i + 2, sample_size=3, lag=2, issue=2020_04_01 + i + 2) for i in range(num_rows)] # <-- the latest issues
```
theres a lot of these that are all the same date......  🥵

Reply:
Refactor'd!